client/pages/fight.jsx

```diff
@@ -55,39 +50,55 @@ export default function Fight(props) {
     setTimeout(() => { setShowCountdown(false); }, 1000);
   }
 
-  function damage(player) {
-    const hit = 20;
-    if (player === 'you') {
-      setYourHp(yourHp - hit);
-    } else if (player === 'opp') {
-      setOppHp(oppHp - hit);
-    }
-  }
-
   // get metaData
-  useEffect(() => {
-    setMetaData(dummyMeta);
-    setYourId(dummyMeta.hostId);
-    setYourUsername(dummyMeta.hostUsername);
-    setOppUsername(dummyMeta.oppUsername);
+  useEffect(async () => {
+    const metaData = await fetch(`/api/game/${gameId}`)
+      .then(res => res.json())
+      .then(result => {
+        return result;
+      });
```

Comment:
If you're using .then method chains to handle Promises, would the async/await still be necessary?